java-extensions/iceberg-metadata-reader/src/main/java/com/starrocks/connector/iceberg/IcebergMetadataScanner.java

```diff
@@ -82,6 +82,9 @@ public class IcebergMetadataScanner extends AbstractIcebergMetadataScanner {
                     "partition",
                     "key_metadata",
                     "split_offsets",
+                    "referenced_data_file",
```

Comment:
this is used for dv，I think we can support read dv later

Reply:
sure